app/jobs/organization_export_job.rb

```diff
@@ -0,0 +1,39 @@
+class OrganizationExportJob < ApplicationJob
+  PART_SIZE = 1_000
```

Comment:
We should double check with analytics that it's cool to send Redshift 50 csvs and 50 copy statements. 

@wrgoldstein: Basically, we're dividing the database into chunks of 1000 rows, exporting those as CSVs and then issuing a copy statement to Redshift for each CSV. The lowest number will start at around 50 CSVs and climb as the database grows. Bearden fires this sync off when new datasets are uploaded, so this shouldn't happen _that often_. This is great for Rails performance. And Redshift seems to be fine with it. But I wanted to draw your attention here in case you know of any problems it could cause.